microcosm_caching/decorators.py

```diff
@@ -118,3 +126,73 @@ def cache(*args, **kwargs) -> Schema:
 
         return cache
     return decorator
+
+
+def invalidates(
+    component,
+    invalidations,
+    cache_prefix,
+    lock_ttl=DEFAULT_LOCK_TTL
+):
+    """
+    Invalidates a set of prescribed keys, based on a combination of:
+        * specified arguments
```

Comment:
is this the right order of the tuple?

Reply:
No, good point. I can make it more structured.